conf/log4j-compression.properties

```diff
@@ -0,0 +1,33 @@
+#-------------------------------------------------------------
```

Comment:
this file need to be moved or removed, 
The file is specific for your experiments.
Optionally we can add it into the scripts/perftest/ohe_checks folder and then you modify your scripts to use it from there.